### ISSUE TITLE
Aborted execution on dotnet TransferUniProtModifications

### ISSUE THREAD

**Issue by Stanley80** [smith-chem-wisc/Spritz#199]
Dear Spritz Developers,
thank you very much for your amazing work.

Today, we successfully installed Spritz in a custom conda environment of a CENTOS server.
When we launched snakemake everything worked fine, but at the step 13 somethings happened.

We report the bottom part of log located in /mnt/datasets/spritz/variants/Homo_sapiens.GRCh38.100.spritz.log.

00:01:12        Predicting variants
00:01:12        Creating xml file /mnt/datasets/spritz/variants/Homo_sapiens.GRCh38.100.protein.xml with 365688 transcripts
00:03:21        done.
00:03:21        Logging
00:03:22        Checking for updates...
00:03:23        Done.
Welcome to TransferModifications!
Analyzing UniProt database data/uniprot/Homo_sapiens.protein.xml.gz and 
Unhandled exception. System.NullReferenceException: Object reference not set to an instance of an object.
   at TransferUniProtModifications.ProteinAnnotation.ParseCodingEffectsToXml(String codingEffectFilePaths, Int32 minPeptideLength, String organism) in /mnt/projects/Spritz/Spritz/TransferUniProtModifications/TransferUniProtModifications/ProteinAnnotation.cs:line 33
   at TransferUniProtModifications.TransferUniProtModifications.Main(String[] args) in /mnt/projects/Spritz/Spritz/TransferUniProtModifications/TransferUniProtModifications/TransferUniProtModifications.cs:line 45
/usr/bin/bash: line 1: 73285 Aborted                 dotnet TransferUniProtModifications/TransferUniProtModifications/bin/Release/netcoreapp3.1/TransferUniProtModifications.dll -x data/uniprot/Homo_sapiens.protein.xml.gz -y /mnt/datasets/spritz/variants/Homo_sapiens.GRCh38.100.protein.xml

It seems to be a bug!!!

Can someone have the solution to this issue?

Thank you very much

**Comment by acesnik** [comment]
Thank you for reporting the bug. Yes, we've had a little trouble with the commandline version as of late (https://github.com/smith-chem-wisc/Spritz/issues/195), and fixing this functionality is the top priority for this project right now. I'll keep you posted!

**Comment by acesnik** [comment]
Hi again! 

I believe this is actually due to the same issue we saw here: https://github.com/smith-chem-wisc/Spritz/issues/201. I believe we have resolved this one by updating mzLib, an underlying library for Spritz. Feel free to give this another shot.

Best regards,

Anthony

**Comment by acesnik** [comment]
I just double-checked that this works on my machine's commandline (macOS 11.1), and it does make it through this step.

**Comment by Stanley80** [comment]
Thank you very much acesnik!!! I'll try it now

**Comment by acesnik** [comment]
Closing this for now. Feel free to reopen if the problem persists.

**Comment by stsour** [comment]
Hello @acesnik !

I would like to reopen this issue. I have just cloned the current Spritz release into my CENTOS server home drive. This is an access-protected server and I went through all the steps for offline computing. Everything worked until step 11. 

I am running into the exact same issue as Stanley80, but I presume that the mzLib update is included in the latest release? For some reason, this is not solving the problem for me. 

Would appreciate some assistance! Thanks.
